CONFIGURATION.md

```diff
@@ -140,3 +140,4 @@ offset.store.method                      |  C  | file, broker    |        broker
 consume.callback.max.messages            |  C  | 0 .. 1000000    |             0 | Maximum number of messages to dispatch in one `rd_kafka_consume_callback*()` call (0 = unlimited) <br>*Type: integer*
 
 ### C/P legend: C = Consumer, P = Producer, * = both
+
```

Comment:
This file is automatically generated but seems to be missing the zstd stuff; make sure you build the source with `make` at least once to let auto-generation kick in.